### PR TITLE
Add helpers for conversation events (PURCHASE-1896)

### PR DESCRIPTION
**Pull request by starsirius**
https://artsyproduct.atlassian.net/browse/PURCHASE-1896

Following up on https://github.com/artsy/cohesion/pull/68, this adds helpers for Conversations events.

Corresponding Force PR in https://github.com/artsy/force/pull/5844.

### CODE REVIEW

File: src/Helpers/Conversations/FocusedOnConversationMessageInput.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { ActionType, FocusedOnConversationMessageInput } from "../../Schema"
+
+export interface FocusedOnConversationMessageInputArgs {
+  impulse_conversation_id: string
```
Comment (eessex) [inline src/Helpers/Conversations/FocusedOnConversationMessageInput.ts:4]: We have been transforming casing to match our js consumers for these args, so would be `impulseConversationId`.

Comment (starsirius) [reply src/Helpers/Conversations/FocusedOnConversationMessageInput.ts]: Ohh nice catch! Fixed in 9afec93.

File: src/Schema/Events/Conversations.ts [hunk 2]
```diff
@@ -14,14 +14,13 @@ import { ActionType } from "."
...
- *    impulse_conversation_id: 198
- *
+ *    impulse_conversation_id: "198"
  *  }
  * ```
  */
 export interface FocusedOnConversationMessageInput {
   action: ActionType.focusedOnConversationMessageInput
-  impulse_conversation_id?: number
+  impulse_conversation_id: string
```
Comment (starsirius) [inline src/Schema/Events/Conversations.ts:23]: Changed this to string to be more generic and required it; in fact, MP returns a string.

**Comment by starsirius** [comment]
Merging this and give it a try! Thanks for review @eessex!

**Comment by artsyit** [comment]
:rocket: PR was released in [`v1.15.0`](https://github.com/artsy/cohesion/releases/tag/v1.15.0) :rocket:

**Pull request merged**